lib/src/tree.dart

```diff
@@ -676,7 +679,7 @@ class MixinDefinition extends Directive {
 
 /** Support a Sass @mixin. See http://sass-lang.com for description. */
 class MixinRulesetDirective extends MixinDefinition {
-  final List<RuleSet> rulesets;
+  final List rulesets;
```

Comment:
This will get fixed further in a follow up which reduces warnings... I'm almost done there. Do you want me to rebase with those changes or does this initial CL look good to you?